### PR TITLE
Add salt.go and request.go

### PR DESCRIPTION
**Pull request by tieubao**

### CODE REVIEW

File: salt.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+)
+
+type Salt struct {
+	Username string
+	Password string
+	Token    string
+}
+
+const (
+	ROOT_URL = "128.199.114.94"
```
Comment (bclermont) [inline salt.go]: ``ROOT_URL`` should be an attribute of ``Salt`` struct

this IP is just a test

Comment (tieubao) [inline salt.go]: ah, thank you

Comment (bclermont) [inline salt.go]: the protocol can also be https

is http client support both https and http?

Comment (tieubao) [inline salt.go]: @bclermont will take a look at it

Comment (tieubao) [inline salt.go]: @bclermont I think net/http package also support https

http://stackoverflow.com/questions/12122159/golang-how-to-do-a-https-request-with-bad-certificate

`resp, err := client.Get("https://example.com")`

Comment (bclermont) [inline salt.go]: perfect

**Comment by bclermont** [comment]
the goal with pull request is to have code review and once it pass review and test it's merged

which had not been the case here

**Comment by bclermont** [comment]
the IP is still hardcoded

don't know what is going on here. i'm giving up

**Pull request merged**